sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/PropagatorConfiguration.java

```diff
@@ -23,6 +25,11 @@ static ContextPropagators configurePropagators(ConfigProperties config) {
     for (String propagatorName : config.getCommaSeparatedValues("otel.propagators")) {
       propagators.add(PropagatorConfiguration.getPropagator(propagatorName));
     }
+
+    for (PropagatorProvider provider : ServiceLoader.load(PropagatorProvider.class)) {
+      propagators.add(provider.get());
+    }
+
     return ContextPropagators.create(TextMapPropagator.composite(propagators));
```

Comment:
somewhere we need to du-dupe this vs. the env var/system prop. Where should that be done?

Reply:
Went ahead and deduped here.